x/emissions/proto/emissions/v1/worker.proto

```diff
@@ -59,9 +60,11 @@ message InferenceForecastBundle {
 message WorkerDataBundle {
   option (gogoproto.equal) = true;
   string worker = 1;
-  InferenceForecastBundle inference_forecasts_bundle = 2;
-  bytes inferences_forecasts_bundle_signature = 3;
-  string pubkey = 4;
+  Nonce nonce = 2;
```

Comment:
these protos reuse numbers, they should not - they should reuse where appropriate, and create new ones